### ISSUE TITLE
Next js 11 build issues (both dev and build)

### ISSUE THREAD

**Issue by wroughtec** [vercel/next.js#26177]
### What version of Next.js are you using?

11.0.0 (& 11.0.1-canary.0)

### What version of Node.js are you using?

14.17.0

### What browser are you using?

NA

### What operating system are you using?

Windows WSL2 (Ubuntu 20.04)

### How are you deploying your application?

NA

### Describe the Bug

On running dev getting the following issue when it first starts to build (has not hit the browser yet):
Files are typescript and has a custom .babelrc file. (appreciate there are a few tickets open that are probably the same issue but hopefully the strack trace will help)
```
Error parsing bundle asset "/.next/static/chunks/webpack.js": no such file
Error parsing bundle asset "/.next/static/chunks/react-refresh.js": no such file
Error parsing bundle asset /.next/static/chunks/pages/_app.js": no such file
Error parsing bundle asset "/.next/static/chunks/pages/_error.js": no such file
Error parsing bundle asset "/.next/static/chunks/amp.js": no such file
Error parsing bundle asset "/.next/static/chunks/polyfills.js": no such file
Error parsing bundle asset "/.next/static/chunks/main.js": no such file

No bundles were parsed. Analyzer will show only original module sizes from stats file.

Webpack Bundle Analyzer saved report to /.next/analyze/client.html
Error parsing bundle asset "/.next/server/webpack-runtime.js": no such file
Error parsing bundle asset "/.next/server/pages/_error.js": no such file
Error parsing bundle asset "/.next/server/pages/_document.js": no such file
Error parsing bundle asset "/.next/server/pages/_app.js": no such file

No bundles were parsed. Analyzer will show only original module sizes from stats file.

Webpack Bundle Analyzer saved report to /.next/analyze/server.html
error - ../../node_modules/next/dist/client/dev/amp-dev.js
SyntaxError: Unexpected token / in JSON at position 525
    at JSON.parse (<anonymous>)
```

Building (none dev) gives the following error:

```
Failed to compile.

../../node_modules/next/dist/client/next.js
SyntaxError: Unexpected token / in JSON at position 525
    at JSON.parse (<anonymous>)


> Build error occurred
Error: > Build failed because of webpack errors
    at /node_modules/next/dist/build/index.js:15:924
    at async Span.traceAsyncFn (/node_modules/next/dist/telemetry/trace/trace.js:6:584)
```

### Expected Behavior

Dev & build not to fail

### To Reproduce

running this command from package.json (which had no issues in v10*):
`cross-env NODE_ENV=development NODE_OPTIONS='--inspect --max_old_space_size=8192' next dev`

**Comment by wroughtec** [comment]
Ok seems to be linked to both:
https://github.com/vercel/next.js/issues/26163
As had some comments in there (again next 10 had no issues)
When comments removed dev worked, however build failed as complained about 

```
logo.svg
TypeError: unsupported file type: undefined (file: undefined)
```

**Comment by MaximeBernard** [comment]
Might be a duplicate of https://github.com/vercel/next.js/issues/26132

**Comment by fitfab** [comment]
it also happens to me on my mac:
macOS Mojave
Version: 10.14.3

<img width="1444" alt="Screen Shot 2021-06-16 at 10 05 03 AM" src="https://user-images.githubusercontent.com/286703/122235939-1485fa00-ce8c-11eb-9531-8f2c11c7a58b.png">

**Comment by gustavpursche** [comment]
Also a duplicate of https://github.com/vercel/next.js/issues/25950

**Comment by wroughtec** [comment]
Hi yes that seems to have resolved my production build issues. Obviously not a long term solution as would be good to take advantage of the static image features but unblocks me for now

**Comment by gustavpursche** [comment]
@wroughtec I agree. There is some ongoing discussion about this here: https://github.com/vercel/next.js/discussions/26167

**Comment by PythonCreator27** [comment]
I (think) that this has been solved in the latest canary, `11.0.0-canary.4`, by #26281. Could someone confirm (after removing `disableStaticImages`)?

**Comment by wroughtec** [comment]
Yep that seems to resolve the build 👍

**Comment by timneutkens** [comment]
There's 2 separate reports in this thread, both of them have been fixed on canary so I'm going to close this issue 👍

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
